cloudslang-runtime/src/main/resources/scripts/cs_replace.py

```diff
@@ -0,0 +1,2 @@
+def cs_replace(str, old_val, new_val, count = -1):
+  return str.replace(old_val, new_val) if count < 0 else str.replace(old_val, new_val, count)
```

Comment:
str should be checked for string

Reply:
@alexbajzat this seems legit, if user want to cast to some number of whatever it is fine to fail, we are not planning to previent user to make mistakes.